### PR TITLE
Add `Project Loved: Six Years / September 2023` news post

### PR DESCRIPTION
**Pull request by Hiviexd**
🎉 

- [x] upload videos to assets
- [x] generate forum posts
- [x] retime/redate if needed

### CODE REVIEW

File: wiki/Community/Project_Loved/en.md [hunk 1]
```diff
@@ -15,14 +15,7 @@ To submit a beatmap for Loved, please visit the [beatmap submission page](https:
...
 
-Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching a specified threshold percentage of "Yes" votes at their polls' end time will be moved to the Loved category:
-
-| Mode | Requirement |
-| :-- | :-- |
-| osu! | 85% |
-| osu!taiko | 80% |
-| osu!catch | 80% |
-| osu!mania | 80% |
+Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching an 80% threshold percentage of "Yes" votes at their polls' end time will be moved to the Loved category:
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:18]:
```suggestion
Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% of "Yes" votes at their polls' end time will be moved to the Loved category:
```

File: news/2023/2023-09-20-project-loved-six-years-september-2023.md [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+
+![](/wiki/shared/news/banners/project-loved-2.jpg)
+
+Welcome or welcome back to another edition of Project Loved! For those who are not yet familiar with who we are and what we do, Project Loved hosts community voting polls each month to decide which beatmaps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). All of the maps are chosen by the [Loved captains](https://loved.sh/team) of their respective game modes. Each beatmap banner in this news post will redirect you to the respective poll, or you can view them all directly on the [Project Loved forum](https://osu.ppy.sh/community/forums/120). The voting process lasts for 10 days, so don't forget to vote! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh).
+
+As we usually do, we would like to thank everyone for your participation in the August votes. If you would like to view the results from the previous round, you may do so by clicking the following links:
+
+- [Results for osu!](https://osu.ppy.sh/community/forums/posts/9254939)
+- [Results for osu!taiko](ahttps://osu.ppy.sh/community/forums/posts/9254938)
```
Comment (WalterToro) [inline news/2023/2023-09-20-project-loved-six-years-september-2023.md:16]:
```suggestion
- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/9254938)
```

Comment (cl8n) [reply news/2023/2023-09-20-project-loved-six-years-september-2023.md:16]: fixed on website, it'll be applied here next time Hivie updates.

File: wiki/Community/Project_Loved/en.md [hunk 3]
```diff
@@ -15,14 +15,7 @@ To submit a beatmap for Loved, please visit the [beatmap submission page](https:
...
 
-Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching a specified threshold percentage of "Yes" votes at their polls' end time will be moved to the Loved category:
-
-| Mode | Requirement |
-| :-- | :-- |
-| osu! | 85% |
-| osu!taiko | 80% |
-| osu!catch | 80% |
-| osu!mania | 80% |
+Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category:
```
Comment (cl8n) [inline wiki/Community/Project_Loved/en.md:18]:
```suggestion
Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category.
```

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**